src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -807,6 +809,19 @@ private void handleOptionAnnotation(final Field field, final int fieldPosition)
                 printOrder = fieldPosition * 1000;
             }
 
+            // log a warning if boundaries are set
+            if (optionAnnotation.maxValue() != Double.POSITIVE_INFINITY) {
+                logger.warn("UNSUPPORTED: maximum allowed value for argument --{} is not forced", field.getName());
```

Comment:
Thanks for adding these. I would remove the phrase "UNSUPPORTED: "  from all four of these, since we really are just issuing a warning. Also, I think these messages should use the argument name, not the field name (sorry if I misled you in the last round of comments). Finally, "forced" should -> "enforced" in these strings.